mods/d2k/rules/structures.yaml

```diff
@@ -79,7 +80,8 @@ construction_yard:
 		FactionImages:
 			atreides: conyard.atreides
 			ordos: conyard.ordos
-			corrino: conyard.corrino
+			smuggler: conyard.ordos
+			mercenaries: conyard.ordos
```

Comment:
I noticed, but 
a) I'd have to override corrino as well, so we'd save only 1 line per structure and not 2, and 
b) it would increase the diff and changing the default increases the risk of regressions.

I'll do it if you insist, but I felt it wasn't worth the additional work.